lib/sounds_player.dart

```diff
@@ -1,13 +1,21 @@
 import 'package:audioplayers/audioplayers.dart';
+import 'package:hot_cold/locator.dart';
 
 class SoundsPlayer {
   final soundsPlayer = AudioPlayer();
+  static const double volumeMultiplier = 0.5;
```

Comment:
since the volume multiplier for each sound is gonna be different, I think it makes more sense to just hardcode it in each `playXSound` function

Reply:
Yeah, fair enough